extension-manifest-v3/src/pages/panel/views/tracker-details.js

```diff
@@ -34,6 +34,27 @@ function showCopyNotification(host) {
   host.querySelector('#gh-panel-company-alerts').appendChild(wrapper);
 }
 
+const requestList = (name, icon, requests) =>
```

Comment:
For me this is again a visual component, so you can avoid nested function in the template.

Reply:
I've inlined it with some code repetition